docs/update.md

```diff
@@ -0,0 +1,37 @@
+# Updating Groups
+
+Inago includes an `update` command, which allows updating
+a group on the cluster.
+
+__Note__: The `update` command only works on sliced groups.
+
+## Usage
+
+Updating a group includes stopping old slices and starting new ones.
+The behavior of the update process is determined by two flags: `--min-alive` and `--max-growth`.
+
+### min-alive
+The `--min-alive` flag defines the minimum number of slices,
+that must be running at any time during the update process.
+
+### max-growth
+The `--max-growth` flag sets the upper limit on how many additional 
+slices may be temporarily started during the update process.
+
+### Update Strategies
+
+Using the above mentioned flags you can enforce various update strategies. We will show this using the `myapp` example from the [Getting Started](getting_started.md) part of this documentation, assuming we have started `n=3` slices of the group.
```

Comment:
I think you can cut a lot of this, and it reads a bit nicer: 'We will show this using the `myapp` example from [Getting Started]`.